code/datums/components/horror_aura.dm

```diff
@@ -0,0 +1,10 @@
+/datum/component/horror_aura/Initialize()
+	if(ismob(parent))
+		RegisterSignal(parent, .proc/aura_effect)
+
+/datum/component/horror_aura/proc/aura_effect()
+	var/atom/movable/AM = parent
+	for(var/mob/living/carbon/human/H in view(7,src))
```

Comment:
if you want to be even fancier you could make this range variable by making it a variable and having /Initialize() be /Initialize(radius) --> src.radius = radius.